documentation/adr/0016-use-double-array-in-memory-for-pixel-data.md

```diff
@@ -26,21 +27,16 @@ Significant performance improvements can be seen in some algorithms when perform
 
 Pixel binning is performed in both the MATLAB and C++ code. These routines do not create a consistent binning resulting from rounding errors in calculated data.
 
```

Comment:
As it stands, I am in doubt whether this is a misprint for "resulting IN rounding errors". Could we rephrase to make it clearer that the inconsistent binning is a result of rounding errors?